app/views/forever_style_guide/sections/components/navigation/_nav_ambassador_dropdown.erb

```diff
@@ -56,33 +56,33 @@
                 </div>
               <% end %>
 
-              <% if current_ambassador.website %>
+              <% if current_ambassador.website.present? %>
                 <div class="col-grid-float-6">
-                  <%= link_to '//' + current_ambassador.website, title: current_ambassador.website, class: 'btn btn-default btn-link btn-action dropdown-ambassador-contact_btn', target: '_blank' do %>
+                  <%= link_to absolute_url(current_ambassador.website), title: current_ambassador.website, class: 'btn btn-default btn-link btn-action dropdown-ambassador-contact_btn', target: '_blank' do %>
```

Comment:
I think we should have the title also match the link exactly, so change 'title: current_ambassador.website' to 'title: absolute_url(current_ambassador.website)'